TMDBApp/TMDBApp/DetailMovieViewController.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  DetailMovieViewController.swift
+//  TMDBApp
+//
+//  Created by Richard Leh on 16/01/2017.
+//  Copyright © 2017 Richard Leh. All rights reserved.
+//
+
+import UIKit
+
+class DetailMovieViewController: UIViewController {
+
+    @IBOutlet weak var scrollView: UIScrollView!
+    @IBOutlet weak var contentView: UIView!
+    
+    @IBOutlet weak var posterImageView: UIImageView!
+    @IBOutlet weak var titleMovieLabel: UILabel!
+    @IBOutlet weak var releaseLabel: UILabel!
+    @IBOutlet weak var genrerLabel: UILabel!
+    @IBOutlet weak var overviewLabel: UILabel!
+    
+    var movie:Movie?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        self.updateUI()
+        self.fillView()
+    }
+    
+    fileprivate func updateUI(){
+        
+        self.title = movie?.title
+        
+        self.view.backgroundColor = UIColor.init(hexString: Colors.defaultDarkBlue.rawValue)
```

Comment:
You have an enum with color constants inside the Constants file. How about creating an UIColor extension instead?